tests/resources/application.yml

```diff
@@ -11,6 +11,9 @@ instrument:
     home: ${instrument.home}shared/Calibration
     sample:
       home: ${instrument.calibration.home}/CalibrantSamples
+      extensions:
+        - cif
+        - json
```

Comment:
This will make CIF files visible as calibrant sample files.  That could cause an issue, since CIF files don't have any of the information a calibrant sample file has

Reply:
If the user tries selecting a CIF from the dropdown

![image](https://github.com/neutrons/SNAPRed/assets/52183986/2cc28e1f-bd30-447d-9bad-5b56a8667e51)